framework/src/postprocessors/AxisymmetricCenterlineAverageValue.C

```diff
@@ -20,6 +20,9 @@ InputParameters
 validParams<AxisymmetricCenterlineAverageValue>()
 {
   InputParameters params = validParams<SideAverageValue>();
+  params.addClassDescription("Computes the integral of an variable on a sideset using an area or "
```

Comment:
Computes the average value of a variable on a sideset located along the centerline of an axisymmetric model